app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision/cloning.rb

```diff
@@ -8,19 +8,17 @@ def prepare_for_clone_task
   end
 
   def start_clone(clone_options)
-    begin
       if request_type == 'clone_to_template'
         make_request_clone_to_template(clone_options)
       elsif sap_image?
         make_request_clone_sap_vm(clone_options)
       else
         make_request_clone(clone_options)
       end
-    rescue IbmCloudPower::ApiError => e
-      raise MiqException::MiqProvisionError, e.response.to_s
-    end
-  rescue IbmCloudPower::ApiError => e
-    raise MiqException::MiqProvisionError, e.response.to_s
+  rescue IbmCloudPower::ApiError => err
+    error_message = JSON.parse(err.response_body)["description"]
```

Comment:
I think it _should_, but that's not documented so seems like a bad idea to assume. Maybe something like this would be better?
```suggestion
    error_message = JSON.parse(err.response_body)["description"] || "no error description"
```

Reply:
I went with `error_message = JSON.parse(err.response_body)["description"] || err.message`, which is the response_body + http status code + headers. So in the case of a missing "description" there will be hopefully be _something_ useful.